chromium_src/content/public/browser/browser_context.h

```diff
@@ -6,7 +6,11 @@
 #ifndef BRAVE_CHROMIUM_SRC_CONTENT_PUBLIC_BROWSER_BROWSER_CONTEXT_H_
 #define BRAVE_CHROMIUM_SRC_CONTENT_PUBLIC_BROWSER_BROWSER_CONTEXT_H_
 
+#define BRAVE_BROWSER_CONTEXT_H_ \
```

Comment:
Maybe use IsOffTheRecord to also define IsTor to avoid the patch?

Reply:
I already tried that, it doesn't work with `virtual` function

Reply:
Builds ok for me. I tried:
<pre>
#define IsOffTheRecord IsTor() const; \
  virtual bool IsOffTheRecord
</pre>

Reply:
I know it builds ok, there was a crash when I use define method. But I just tried it now, it doesn't crash anymore so I guess there must be something else fix the issue with the following commits.
Anyway, I will use define instead of patch

Reply:
fixed in d38e8f5db1429052e1130c193f7c3562bbfbc1e1